source/optional_features/moto_payments/index.html.md.erb

```diff
@@ -0,0 +1,44 @@
+---
+title: Take payments over the phone ('MOTO' payments)
+weight: 200
+---
+
+# Take payments over the phone (‘MOTO’ payments)
+
+We can set up your service so you can fill in GOV.UK Pay payment pages with your users’ details.
+
+This means you can take payments:
+
+- over the phone
+- by post
+
+These payments are also known as Mail Order Telephone Order (MOTO) payments.
+
+After we set up your service, you'll no longer be able to create payments where users fill in their details themselves. [Create a separate service](https://selfservice.payments.service.gov.uk/my-services/create) if you need to do this.
```

Comment:
MOTO gateway accounts do allow 'normal' user-led online payments

Reply:
Ah sorry, I might have misunderstood this.
Would you be able to take 'normal' payments on Worldpay, ePDQ and Stripe? Or just Stripe?

Reply:
Yeah I think there's some confusion here. If Stripe is being used, MOTO and normal payments can be made. However as we discussed yesterday, for Worldpay and ePDQ where services can take MOTO payments by virtue of using a MID, we decided that it was best to use a gateway account for one of MOTO payments or normal payments. The reason is that if we want to enable gateway accounts using Worldpay and ePDQ to take MOTO and normal payments, we'd have to do work to deal with two MIDs on a single gateway account.

Reply:
we did this so that services can have MOTO and non-MOTO payments within one account, so reporting can be in one place.